StripeFinancialConnections/StripeFinancialConnections/Source/Web/FinancialConnectionsWebFlowViewController.swift

```diff
@@ -292,7 +292,10 @@ extension FinancialConnectionsWebFlowViewController {
         return components
             .queryItems?
             .first(where: { $0.name == key })?
-            .value?.removingPercentEncoding
+            .value?
+            .removingPercentEncoding?
+        // backend can return "+" instead of a more-common encoding of "%20" for spaces
+            .replacingOccurrences(of: "+", with: " ")
```

Comment:
non-blocking, but any concern with banks that legitimately have a + as part of their name? seems only a matter of time before we have banks hopping on the "plus" branding :)  "Chase+"

Reply:
yea its a good point; I looked at various institutions we have and this should nearly never happen (if at all), and if it does this is used just for display so this seems like an OK decision _for now_ (not perfect! but justifiable)